src/app/shared/components/filters-list/searchbar/searchbar.component.ts

```diff
@@ -78,6 +84,16 @@ export class SearchbarComponent implements OnInit, OnDestroy {
     this.performSearch();
   }
 
+  public handleInvalidCharacter(value: string): void {
+    const validValue = value?.replace(/[^A-Za-zА-Яа-я0-9`.,№"']/g, '');
+    if (validValue !== value) {
+      this.searchValueFormControl.setValue(validValue);
+      this.invalidCharacterDetected.emit();
```

Comment:
Can we use app-validation-hint instead of triggering an error to the parent component?

Reply:
I don't think so, because when we use app-validation-hint in this component it appears directly under search input and causes everything inside this component to shift.

Reply:
I don't agree with doing different types of validation for different form fields. We should use a consistent approach throughout the application to ensure the same experience and interface. You can try adjust search bar size, leaving a space for the error message there